alf/algorithms/algorithm.py

```diff
@@ -239,6 +239,8 @@ def _preload_checkpoint(self):
 
             self.load_state_dict(stat_dict, strict=True)
             self._checkpoint_pre_loaded = True
+            logging.info(
```

Comment:
use `common.info` will be better highlighted

Reply:
switched to common.info